Client/game_sa/CHandlingManagerSA.cpp

```diff
@@ -215,82 +222,82 @@ CHandlingManagerSA::~CHandlingManagerSA()
     }
 }
 
-eHandlingProperty CHandlingManagerSA::GetPropertyEnumFromName(const std::string& strName) const
+eHandlingProperty CHandlingManagerSA::GetPropertyEnumFromName(const std::string& strName) const noexcept
 {
     const auto it = m_HandlingNames.find(strName);
     return it != m_HandlingNames.end() ? it->second : HANDLING_MAX;
 }
 
-CHandlingEntry* CHandlingManagerSA::CreateHandlingData()
+CHandlingEntry* CHandlingManagerSA::CreateHandlingData() const noexcept
 {
-    return new CHandlingEntrySA;
+    return new (std::nothrow) CHandlingEntrySA;
 }
 
-CFlyingHandlingEntry* CHandlingManagerSA::CreateFlyingHandlingData()
+CFlyingHandlingEntry* CHandlingManagerSA::CreateFlyingHandlingData() const noexcept
 {
-    return new CFlyingHandlingEntrySA;
+    return new (std::nothrow) CFlyingHandlingEntrySA;
 }
 
-CBoatHandlingEntry* CHandlingManagerSA::CreateBoatHandlingData()
+CBoatHandlingEntry* CHandlingManagerSA::CreateBoatHandlingData() const noexcept
 {
-    return new CBoatHandlingEntrySA;
+    return new (std::nothrow) CBoatHandlingEntrySA;
 }
 
-CBikeHandlingEntry* CHandlingManagerSA::CreateBikeHandlingData()
+CBikeHandlingEntry* CHandlingManagerSA::CreateBikeHandlingData() const noexcept
 {
-    return new CBikeHandlingEntrySA;
+    return new (std::nothrow) CBikeHandlingEntrySA;
 }
 
-const CHandlingEntry* CHandlingManagerSA::GetOriginalHandlingData(eVehicleTypes eModel) const
+const CHandlingEntry* CHandlingManagerSA::GetOriginalHandlingData(std::uint32_t uiModel) const noexcept
```

Comment:
Dont use hungarian notation

Reply:
How do I guess you want me to use the name 'model'?

Reply:
Yes